storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc

```diff
@@ -0,0 +1,39 @@
+[id="persistent-storage-csi-google-cloud-file"]
+= Google Filestore CSI Driver Operator
+include::_attributes/common-attributes.adoc[]
+:context: persistent-storage-csi-google-cloud-file
+
+toc::[]
+
+[id="persistent-storage-csi-google-cloud-file-overview"]
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Google Filestore Storage.
+
+:FeatureName: Google Filestore CSI Driver Operator
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+To create CSI-provisioned PVs that mount to Google Filestore Storage assets, {product-title} installs the Google Filestore CSI Driver Operator and the Google Filestore CSI driver in the `openshift-cluster-csi-drivers` namespace.
```

Comment:
OCP doesn't really installs the GCP Filestore operator (like it does with the GCP PD operator), the user does.

Reply:
Done.